openstudio-standards/lib/openstudio-standards/prototypes/Prototype.utilities.rb

```diff
@@ -145,14 +145,8 @@ def hspf_to_cop_heating_no_fan(hspf)
 # @param [Double] COP
 # @return [Double] Seasonal Energy Efficiency Ratio
 def cop_to_seer(cop)
-  seer = nil
-
-  # First convert COP to EER
-  eer = cop_to_eer(cop)
-
-  # Next convert from EER to SEER
-  delta = 1.1088**2 - 4.0 * 0.0182 * eer
-  seer = (-delta**0.5 + 1.1088) / (2.0 * 0.0182)
+  delta = 0.3796**2 - 4.0 * 0.0076 * cop
```

Comment:
@asparke2, This new method 'cop_to_seer' is based on method 'seer_to_cop_cooling_no_fan'. It uses then the same methodology to get cop from seer. So the reference listed in the existing method still applies in the new method.